cmd/mattermost/commands/channel.go

```diff
@@ -554,3 +565,39 @@ func renameChannelCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func searchChannelCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	teamID, term := parseChannelArg(args[0])
+	foundChannelList, err2 := a.SearchChannels(teamID, term)
```

Comment:
I think I would expect to search channels in all teams, unless you pass a flag `--team` with the team name or id.

I thinking about something like:

```
mattermost channel search whatever # <-- Search for "whatever" term in all teams (in a loop, probably).
mattermost channel search --team dev whatever # <-- Search for "whatever" term in the team with name `dev` or id `dev`
mattermost channel search --team jdlfndalkn34242 whatever # <-- Search for "whatever" term in the team with name `jdlfndalkn34242` or id `jdlfndalkn34242`
```

Reply:
Great, thanks! I needed these requirements. I work on this and will update the PR.